app/src/app_common.cc

```diff
@@ -190,7 +190,7 @@ struct AppData {
 // Tracks library registrations.
 class LibraryRegistry {
  private:
-  LibraryRegistry() {}
+  LibraryRegistry() : is_common_libraries_registered(false) {}
```

Comment:
nit: what about naming this, and the accessors, "Library" instead of "Libraries", as IsCommonLibraryRegistered is more correct?

Reply:
Done